src/AppBundle/Entity/Report/ProfServiceFee.php

```diff
@@ -0,0 +1,291 @@
+<?php
+
+namespace AppBundle\Entity\Report;
+
+use AppBundle\Entity\Report\Traits\HasReportTrait;
+use JMS\Serializer\Annotation as JMS;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @JMS\Discriminator(field = "fee_type_id", map = {
+ *    "current": "AppBundle\Entity\Report\ProfServiceFeeCurrent"
+ * })
+ */
+abstract class ProfServiceFee
+{
+    const TYPE_ASSESSED_FEE = 'assessed';
+    const TYPE_FIXED_FEE = 'fixed';
+
+    const TYPE_PREVIOUS_FEE = 'previous';
+    const TYPE_CURRENT_FEE = 'current';
+    const TYPE_ESTIMATED_FEE = 'estimated';
+
+    use HasReportTrait;
+
+    /**
+     * @JMS\Type("integer")
+     * @JMS\Groups({"prof-service-fees"})
+     *
+     * @var int
+     */
+    private $id;
+
+    /**
+     * Hold service type
+     *
+     * If the order or any key is added, update the ReportControllerTest, hardcoded on position and number
+     *  in order to keep it simple
+     *
+     * @var array
+     */
+    public static $serviceTypeIds = [
+        // id => hasMoreDetails
+        'annual-report' => false,
+        'annual-management-interim' => false,
+        'annual-management-final' => false,
+        'appointment' => false,
+        'conveyancing' => false,
+        'tax-returns' => false,
+        'trust-applications' => false,
+//        'litigation' => true,
+//        'specialist-advice' => true,
+//        'statutory-wills' => false,
+        'other-costs' => false
+    ];
+
```

Comment:
@elvisciotti Let's just delete the ones we don't need. Also why is the same list of service types in the API?

Reply:
I cleaned things up